components/dropdown/Dropdown.jsx

```diff
@@ -30,6 +31,18 @@ class Dropdown extends React.Component {
     up: false
   };
 
+  componentWillUpdate (prevState, nextState) {
+    if (!prevState.active && nextState.active) {
+      events.addEventsToDocument({click: this.handleDocumentClick});
```

Comment:
I think here a remove event listener is missing, isn't it? This would add as many event listeners to the document as times you open the dropdown. In other components we added the event listener at mount and removed at unmount. I know this means having an event listener for each component but definitively it should be removed when there is not needed anymore.

Thanks for the PR, I'll be happy to merge if you add the remove :)